spec/system/tech_shop_spec.rb

```diff
@@ -1,8 +1,10 @@
+require 'support/async_helper'
 require 'support/models/product'
 
 describe 'Tech shop' do
   before(:all) do
-    Product.clear_index!(true)
```

Comment:
Funnily enough, this never worked! Product has indexing disabled and in cases where indexing is disabled clear_index does nothing.

However, this did not present an issue since this one describe block was the only thing using product. When this runs after the recently-added multi search specs where I also used product, many of the expectations fail :rofl: 

I found out the hard way as you can imagine.